src/main/java/net/dv8tion/jda/client/JDAClient.java

```diff
@@ -168,5 +169,20 @@
      *         {@link net.dv8tion.jda.client.entities.AuthorizedApplication AuthorizedApplication}
      *         <br>The Application behind the provided id.
      */
-    RestAction<AuthorizedApplication> getAuthorizedApplicationById(String id);    
+    RestAction<AuthorizedApplication> getAuthorizedApplicationById(String id);   
+    
+    
+    
+    /**
+     * Opens a new Group DM between this account and the users specified by the users param
+     *
+     * @param  users The users to open the Group DM with
+     * 
+     * @throws IllegalArgumentException If the user array contains less than 2 user objects or more than 9 user objects
+     *
+     * @return {@link net.dv8tion.jda.core.requests.RestAction RestAction} - Type: 
+     *         {@link net.dv8tion.jda.client.entities.Group Group}
+     *         <br>The new Group DM
+     */
+    RestAction<Group> createGroupDM(ArrayList<User> users);
```

Comment:
New line descriptions of parameters and throwables.
Don't specifically ask for an `ArrayList` a `Collection` should be fine.